project/app/api/covid_score.py

```diff
@@ -0,0 +1,108 @@
+from sodapy import Socrata
+
+# Execute a request to the CDC Covid API
+def req_cdc_covid_dat(state, dte):
+    """
+    req_cdc_covid_dat makes an HTTP GET request to the 
+    CDC Covid API and accepts a response
+
+    Parameters:
+        "state": the targeted state (e.g. "CA")
+        "date":  the date of the covid data requested
+
+    Returns:
+        "ok":       boolean indicating a successful request
+        "new_case": new cases logged in the API response
+        "data":     copy of the CDC API response
+        "error":    error message (if applicable)  
+    """
+    # Define a return object
+    ret_dict = {
+        "ok": False,
+        "new_case":   None,
+        "data":       None,
+        "error":      "an error occurred"
```

Comment:
Maybe error should be none too? If no error found, nothing to do.

Reply:
https://www.latimes.com/science/story/2020-11-11/covid-19-risk-thanksgiving-2020-what-to-expect - or something similar to this "No or missing data"?